front/style/_forms.scss

```diff
@@ -61,3 +61,24 @@
         }
     }
 }
+
+.login {
+    &--box {
+        margin: 20px auto 0;
+        background-color: white;
+
+        &--error {}
```

Comment:
I tend to agree to not add something useless, especially as we won't necessarly change it in the future.